example/session/src/main/java/org/apache/iotdb/VectorSessionExample.java

```diff
@@ -217,9 +239,12 @@ private static void insertTabletWithAlignedTimeseriesMethod1()
     List<IMeasurementSchema> schemaList = new ArrayList<>();
     schemaList.add(
         new VectorMeasurementSchema(
-            new String[] {"s1", "s2"}, new TSDataType[] {TSDataType.INT64, TSDataType.INT32}));
+            "vector",
+            new String[] {"s1", "s2"},
+            new TSDataType[] {TSDataType.INT64, TSDataType.INT32}));
 
-    Tablet tablet = new Tablet(ROOT_SG1_D1, schemaList);
+    Tablet tablet = new Tablet(ROOT_SG1_D1 + ".vector", schemaList);
+    tablet.setAligned(true);
```

Comment:
@qiaojialin I remember we discussed before. We may use normal MeasurementSchema here. Like the code I shows below.
```java
List<IMeasurementSchema> schemaList = new ArrayList<>();
schemaList.add(
    new MeasurementSchema("s1",TSDataType.INT64));
schemaList.add(
    new MeasurementSchema("s2",TSDataType.INT32));

Tablet tablet = new Tablet(ROOT_SG1_D1 + ".vector", schemaList);
tablet.setAligned(true);
```

Reply:
This will be refactored by Zhao Xin later.